### DISCUSSION TITLE
关于mac电脑终端关闭后网络dav失败问题

### DISCUSSION THREAD

**Discussion by Sycun** [AlistGo/alist#7749] [Q&A]
使用webdav的时候必须打开终端窗口使用alist server，关闭终端窗口以后服务也会断开，有什么办法解决吗
![WX20241230-162306@2x](https://github.com/user-attachments/assets/81fca7ab-a428-4af6-8ed5-48f3f4821eb9)

**Comment by pongfcnkl** [comment]
最简单的就是在命令里加nohup

完善一点就自己谷歌或者ai 问launchd使用教程
